tests/VMelnalksnis.PaperlessDotNet.Tests.Integration/Documents/DocumentClientTests.cs

```diff
@@ -94,6 +95,21 @@ public async Task Create()
 
 		updatedDocument.Title.Should().Be(update.Title);
 
+		var documentDownload = await Client.Documents.Download(id, original: true);
+
+		documentDownload.ContentType.Should().Be("text/plain");
+		documentDownload.FileName.Should().Contain("Lorem Ipsum");
+
+		using (var stream = documentDownload.Content)
+		{
+			Assert.IsNotNull(stream);
```

Comment:
This assertion is redundant, no null warnings were suppressed in the download method.